openhcl/virt_mshv_vtl/src/processor/snp/mod.rs

```diff
@@ -1349,8 +1394,7 @@ impl UhProcessor<'_, SnpBacked> {
                 // Receipt of a virtual interrupt intercept indicates that a virtual interrupt is ready
                 // for injection but injection cannot complete due to the intercept. Rewind the pending
                 // virtual interrupt so it is reinjected as a fixed interrupt.
-
-                // TODO SNP: Rewind the interrupt.
+                self.rewind_offloaded_interrupt(dev, entered_from_vtl, false);
                 unimplemented!("SevExitCode::VINTR");
```

Comment:
Would this unimplemented go away?

Reply:
I wanted to leave it until we've actually seen one of these and can be sure nothing else is needed here. But yes in theory all that's needed is deleting the unimplemented.